src/main/java/com/team7419/TalonFuncs.java

```diff
@@ -20,6 +20,5 @@ public static void setPIDFConstants(int slot, BaseTalon talon, double kP, double
 		talon.config_kI(slot, kI, 0);
 		talon.config_kD(slot, kD, 0);
 		talon.config_kF(slot, kF, 0);
-	}
-
+    }
 }
```

Comment:
I mean everything else looked fine until I saw this file. I'm not too sure about this change.